inputFiles/poromechanics/StabilizedPoroStaircase.xml

```diff
@@ -0,0 +1,299 @@
+<?xml version="1.0" ?>
```

Comment:
Please use `xml` extension and not `txt`.

Reply:
We removed this xml from the PR because this integrated test was merged some time ago in another PR.